doc/ref/gappkg.xml

```diff
@@ -27,26 +17,36 @@ integrated into the &GAP; help system.
 All  &GAP;  users   who  develop  new  code  are   invited  to  share
 the  results  of their  efforts  with  other  &GAP; users  by  making
 the  code  and its  documentation  available  in  form of  a  package.
-Information how  to do  this is  available from  the &GAP;  Web pages
+Information how  to do  this is  available from  the &GAP; website
 (<URL>https://www.gap-system.org</URL>)
 and in the &GAP; package <Package>Example</Package>
 (see <URL>https://www.gap-system.org/Packages/example.html</URL>).
 There  are  possibilities  to get  a  package  distributed together  with
 &GAP;  and it  is possible  to submit  a package  to a formal refereeing
 process.
 <P/>
-In this chapter we describe how to use existing packages.
+In this chapter first we describe how to use existing packages,
+and then provide guidelines for writing a &GAP; package.
 
 
 <!-- %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% -->
 <Section Label="Installing a GAP Package">
 <Heading>Installing a GAP Package</Heading>
 
-Before a  package can be  used it must  be installed. With  a standard
-installation of  &GAP; there should  be all currently redistributed with 
-&GAP; packages already available. But since &GAP; packages are released 
-independently of the main  &GAP; system  it  may be  sensible to  upgrade  
-or install new packages between upgrades of your &GAP; installation.
+Before a  package can be  used it must  be installed. A standard installation
```

Comment:
Surely the average (or casual) GAP user just wants to load a package.  So include here the material in the LoadPackage subsection which refers to this, and move to a later section all the stuff about automatic loading.